server/routes.js

```diff
@@ -35,6 +35,7 @@ function configureRoutes(app) {
   app.post('/pa/get-sensor-clients', pa.validateGetSensorClients, googleHelpers.paAuthorize, pa.handleGetSensorClients)
   app.post('/pa/sensor-twilio-number', pa.validateSensorPhoneNumber, googleHelpers.paAuthorize, pa.handleSensorPhoneNumber)
   app.post('/pa/message-clients', pa.validateMessageClients, googleHelpers.paAuthorize, pa.handleMessageClients)
+  app.post('/pa/check-database-connection', pa.validateCheckDatabaseConnection, googleHelpers.paAuthorize, pa.handleCheckDatabaseConnection)
```

Comment:
What do you think about making the route more generic as suggested in the article, something like `/pa/health` or `pa/check-health`? 

I'm OK either way. What you've written is accurate for what it's doing now and if it were extended in the future then the route could change at that time too.

Reply:
I was actually struggling to decide on an appropriate name for this route, so I appreciate your thoughts on this! I like `pa/health` so I'll update it to that.

Reply:
:+1: 

As they say, the two hardest problems in software engineering:
1. Naming things
2. Cache invalidation
3. Off-by-one errors